torch/_meta_registrations.py

```diff
@@ -133,7 +133,7 @@ def meta_angle(self):
         _, result_dtype = elementwise_dtypes(
             self, type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT
         )
-    return self.new_empty(self.size(), dtype=result_dtype)
+    return torch.empty_like(self, dtype=result_dtype)
```

Comment:
What is the difference between the two?

Reply:
torch.empty_like inherits other meta from self, including stride, device, memroy_layout ect... 
We should use torch.empty_like whenever appropriate